src/Build/Logging/FancyLogger/FancyLogger.cs

```diff
@@ -70,6 +67,12 @@ void eventSource_ProjectStarted(object sender, ProjectStartedEventArgs e)
             projects[id] = node;
             // Log
             node.Log();
+            // Update footer
+            FancyLoggerBuffer.FooterText = ANSIBuilder.Alignment.SpaceBetween(
+                $"Finished projects: {ANSIBuilder.Graphics.ProgressBar(FinishedProjects/StartedProjects)} {FinishedProjects}/{StartedProjects}",
```

Comment:
Let's not use the word "projects" here. It's confusing because it's not what users expect by project, which is "a project file". We can have many requests to build a project (several per ProjectReference, plus inner and outer builds for multitargeted projects, plus the restore info-collection) that show up in this number, but it's weird to have 800+ "projects" in a build of MSBuild which has 31 actual projects.

In fact I don't think I'd show the count numbers anywhere, and just show the bar with "Approximate progress:" or similar.

Reply:
Okay. I will change it to say "Build progress (approx.)". Should we also add a percentage after the progress bar? With big denominators the progress bar might appear frozen

Reply:
No objection to a percentage but I don't know if it's necessary.